tendrl/commons/objects/cluster/atoms/delete_cluster_details/__init__.py

```diff
@@ -10,6 +10,15 @@ def __init__(self, *args, **kwargs):
         super(DeleteClusterDetails, self).__init__(*args, **kwargs)
 
     def run(self):
+        logger.log(
+            "info",
+            NS.publisher_id,
+            {
+                "message": "Deleting cluster details."
```

Comment:
Doesn't the atom executor code says executing this atom already? Dow we need this log again?

Reply:
It is not giving for cluster details, that is why it needs to be added.